docs/index.rst

```diff
@@ -68,6 +68,17 @@ Example:
   endpoint: localhost:5000
 
 
+.. Note::
+
+   Ensure that Apache listens on specified port.
+
+
+.. Note::
+   Path specified in `data_dir` should be a shared mount point between Crane and Pulp,
```

Comment:
s/Pulp,/Pulp./

s/where Pulp would have w-r and Crane read-only access./This mount point should be readable and writable by Pulp. Crane only needs read access to this mount./